### ISSUE TITLE
Entropy information for NMR entries. 

### ISSUE THREAD

**Issue by kumar-physics** [eppic-team/eppic#121]
eppic-web has many useful additional features like sequence homologs, entropy potato, etc. There are available only for x-ray entries. Why not we provide these information for NMR entries as well. Right now I see no information for NMR entries, not even the sequence.  It would be good to provide these details for all entries, so NMR community also get benefited from eppic-web.

**Comment by josemduarte** [comment]
That's right, we don't try to calculate alignments for NMR entries when they have no interfaces. Those that have interfaces (e.g. 2mps) do have alignments and entropy potatoes.

This would be a good feature to have so that we cover the whole PDB with alignments. I'll add this to the 3.1 milestone.
